src/BloomBrowserUI/bookEdit/bloomField/bloomField.ts

```diff
@@ -154,10 +155,30 @@ class BloomField {
         e.preventDefault();
     }
 
+    // Since embedded images come before the first editable text, going to the beginning of the field and pressing Backspace moves the current paragraph into the caption. Sigh.
+    private static PreventBackspaceAtStartFromMovingTextIntoEmbeddedImageCaption(field: HTMLElement) {
+        $(field).keydown(e => {
+            if (e.which == 8 /* backspace*/) {
+                var sel = window.getSelection();
+                //Are we at the start of a paragraph?
+                if (sel.anchorOffset == 0
+                    //Are we in the first paragraph? 
+                    //Embedded image divs come before the first editable paragraph, so we look at the previous element and
+                    //see if it is one those. Anything marked with bloom-preventRemoval is probably not something we want to
+                    //be merging with.
+                    && $(sel.anchorNode).closest('P').prev().hasClass('bloom-preventRemoval')) {
+                        e.stopPropagation();
```

Comment:
If I'm reading this right, it will undesirably prevent backspace from (a) deleting a range of text with anchor right before an embedded image, (b) deleting a range at the start of a span within a paragraph following an image, and possibly (c) deleting the character before a span within a paragraph following an image. (i.e., if the selection is an IP at offset 0 in the span).